src/main/java/net/pms/formats/PCX.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Universal Media Server, for streaming any media to DLNA
+ * compatible renderers based on the http://www.ps3mediaserver.org.
+ * Copyright (C) 2012 UMS developers.
+ *
+ * This program is a free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2
+ * of the License only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package net.pms.formats;
+
+
+public class PCX extends ImageBase {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public Identifier getIdentifier() {
+		return Identifier.PCX;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public String[] getSupportedExtensions() {
+		return new String[] { "pcx", "dcx" };
+	}
+
+	@Override
+	public String mimeType() {
+		return "image/x-pcx";
```

Comment:
@Sami32 What is deprecated by IANA and what is generally best understood isn't always the same thing, in my experience people tend to use the "x-" version a lot in general (not for PCX in particular). But, I'll change it - what is the best to use is guessing anyway, and in this case it really doesn't matter - I doubt we will find a renderer that accepts PCX via DLNA, so we'll have to transcode it to some other format anyway and use that mime type.